executor/builder.go

```diff
@@ -3900,6 +3900,15 @@ func (b *executorBuilder) buildIndexLookUpReader(v *plannercore.PhysicalIndexLoo
 	}
 
 	if is.Index.Global {
+		tmp, _ := b.is.TableByID(ts.Table.ID)
+		tbl := tmp.(table.PartitionedTable)
```

Comment:
Please use a bit more defensive style, even if it should be impossible for errors, so something like:
```
tmp, err := b.is.TableByID(ts.Table.ID)
if err != nil {
  return err
}
if tbl, ok := tmp.(table.PartitionedTable) ; ok {
  ...
} else {
  return <Error about internal error?>
}
```